src/app/profile/page.tsx

```diff
@@ -6,16 +6,21 @@ import { ThemeToggler } from '@/components/ui/theme-toggler'
 
 export default function Profile() {
   return (
-    <AuthContent>
-      <ImageCard className="-my-[4.375rem] hidden sm:flex py-32 px-[3.25rem]">
+    <AuthContent className="profile-page bg-white md:bg-gradient-blue-white">
+      <h3 className="text-lg font-medium text-center py-[0.4375rem] md:hidden">
+        Profile
+      </h3>
+      <ImageCard className="p-0 bg-none -mt-3 md:mt-0 md:-my-[4.375rem] md:py-32 md:px-[3.25rem] md:bg-gradient-purple-blue">
         <FileInput />
       </ImageCard>
-      <article>
-        <header className="flex justify-between pb-2 mt-8">
+      <article className="flex justify-center md:block">
+        <header className="hidden justify-between pb-2 mt-8 md:flex">
           <h1 className="title-lg">Profile</h1>
-          <ThemeToggler />
+          <div className="hidden md:block">
+            <ThemeToggler />
+          </div>
         </header>
-        <ProfileForm className="mt-5" />
+        <ProfileForm className="grow mt-[2.1875rem] md:mt-5" />
```

Comment:
Here 1px is not a huge loss, we can avoid arbitrary value

```suggestion
        <ProfileForm className="grow mt-9 md:mt-5" />
```